src/RecipeBot.Domain/Models/RecipeTagsModelWrapper.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (C) 2022 Dennis Tang. All rights reserved.
+//
+// This file is part of RecipeBot.
+//
+// RecipeBot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using Common.Utils;
+using RecipeBot.Domain.Data;
+using RecipeBot.Domain.Properties;
+
+namespace RecipeBot.Domain.Models;
+
+/// <summary>
+/// Wrapper for the <see cref="RecipeTagsModel"/>.
+/// </summary>
+public class RecipeTagsModelWrapper
```

Comment:
Is there still a factual difference between a RecipeTagsModel and the Wrapper since the model is just a collection of tags now?

Reply:
The goal of the RecipeTagsModel is to separate the category and the tags to prevent redundant data. The associated category and tags are combined together within the RecipeModel.

However, this does mean that some refactoring is still needed in the RecipeEmbedFactory